Xwt/Xwt/ColorPicker.cs

```diff
@@ -120,7 +120,7 @@ protected virtual void OnColorChanged (EventArgs args)
 		}
 	}
 
-	class DefaultColorPickerBackend: XwtWidgetBackend, IColorPickerBackend
+	public class DefaultColorPickerBackend: XwtWidgetBackend, IColorPickerBackend
```

Comment:
If there are a11y naming to be done it makes more sense to do them here directly instead of letting the client do it.

Reply:
Here's how I use this: https://github.com/xamarin/designer/blob/207acc07d5e54edd3a4419247b7a740d581300b0/Xamarin.Designer.Android/Xamarin.AndroidDesigner/Xamarin.AndroidDesigner.Selectors/GridOverlaySelector.cs#L255

Note that the a11y name here for the button that launches the color picker comes from the tooltip text for us, which needs to come from the client as it's client specific. There might be a more ideal way to handle this, supporting setting the Accessible name on the frontend ColorPicker instead of the special backend-which-lives-in-the-frontend DefaultColorPickerBackend, but it seemed a bit tricky to plumb thru the Accessible in that special case & this worked fine.

Separately, though, see my subsequent changes in this PR for `ColorSelector.cs`, where I do add a11y names to Xwt for the color related UI (e.g, the RGB and HSL editors).